### ISSUE TITLE
cli tool

### ISSUE THREAD

**Issue by mafintosh** [hypermodules/gh-release-assets#2]
i was just looking for something exactly like this! i need to use it from the command line though. would be cool if this supported command line usage :+1:

**Comment by bcomnes** [comment]
Via [gh-release](https://ghub.io/gh-release)

**Comment by bcomnes** [comment]
https://github.com/ngoldman/gh-release

**Comment by paulcpederson** [comment]
@mafintosh yes, meant to be used via gh-release. Although I guess I could write a quick cli for it to do just this component. That's a pretty good idea!

**Comment by paulcpederson** [comment]
Just to make sure, you mean you want to upload something to an existing release _without_ creating a new release, right?

**Comment by maneetgoyal** [comment]
Will be nice to have 👍. If the CLI comes with **regex support** for those asset paths, it will be a nice addition as well. Also, suggest that the CLI can read the **GitHub token from some environment variable** to prevent hardcoding the token anywhere in the release script.

Here's some info on my use-case:

- I am using Lerna to manage a monorepo.
- Lerna provides CLI support to publish new releases/tags on GitHub.
- But it is [lacking the support to release the assets](https://github.com/lerna/lerna/issues/2743) alongside making a new release.
- Once a new release is made, Lerna updates its `lerna.json` file with the latest version number.
- Hoping that this version number could be used to figure out the `upload_url`.
- Using that URL, I wish to upload some assets.

Will start trying out using the current gh-release-assets API to do it in the meantime.

**Comment by bcomnes** [comment]
Only existing solution I can think of would be to use shell expansion and join the results with commas.  Otherwise adding [glob](http://ghub.io/glob) expressions to [gh-release](https://ghub.ip/gh-release)'s assets flag would be the best way to do this I imagine.   This wouldn't be too hard, but low priority personally so I will unlike work on that any time soon.  I would consider a PR.

**Comment by bcomnes** [comment]
You may also consider using a github action: 

https://github.com/actions/upload-release-asset

But that is actions specific.

**Comment by maneetgoyal** [comment]
Thanks for the suggestion @bcomnes. I was able to work with the current programmatic API to upload the assets. (Used octokit/rest to get the `upload_url` [corresponding to a given tag/version-number](https://developer.github.com/v3/repos/releases/#get-a-release-by-tag-name)). It works fine so the CLI support is a low priority for me too.

**Comment by ungoldman** [comment]
Hi @mafintosh, cli tool published in latest release: https://github.com/ungoldman/gh-release-assets/releases/tag/v3.0.0

sorry for the wait
